### ISSUE TITLE
Correctly detect timeouts

### ISSUE THREAD

**Issue by matthewbauer** [NixOS/ofborg#160]
Ofborg has a timeout which is frequently hit:

```
building of '/nix/store/8x54mjabi6mrpji2y2326zj2zvgkldhz-nodejs-9.11.1.drv' timed out after 1800 seconds
```

It shouldn't report these as "failures". "settings.buildTimeout" is set to 1800 which should be okay for us but it's hit fairly frequently.

**Comment by 7c6f434c** [comment]
EDIT: oops, I fail at search

This was probably discussed on IRC/#nixos-borg but not in the issues

**Comment by grahamc** [comment]
This was resolved by #206

**Comment by matthewbauer** [comment]
Awesome!! Great work
